src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/VSLSPRangeExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    internal static class VSLSPRangeExtensions
+    {
+        public static TextSpan AsTextSpan(this Range range, SourceText sourceText)
```

Comment:
Copied from [`Microsoft.AspNetCore.Razor.LanguageServer`](https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RangeExtensions.cs#L73-L88) implementation for `OmniSharp.Extensions.LanguageServer.Protocol.Models.Range` (this is for `Microsoft.VisualStudio.LanguageServer.Protocol.Range`). They do not share a common interface